sbt-web-scalajs-bundler/src/sbt-test/sbt-web-scalajs-bundler/play/server/src/main/scala/example/ExampleController.scala

```diff
@@ -6,26 +6,19 @@ import play.twirl.api.StringInterpolation
 class ExampleController(cc: ControllerComponents) extends AbstractController(cc) {
 
   val index = {
-    val scriptUrl = bundleUrl("client")
     val result =
       Ok(
         html"""<!doctype html>
         <html>
           <head></head>
           <body>
             <div>App is not loaded.</div>
-            <script src="$scriptUrl"></script>
+            @scalajs.html.scripts("client", routes.Assets.versioned(_).toString, name => getClass.getResource(s"/public/$$name") != null)
```

Comment:
Shouldn’t we do this?

```suggestion
            @scalajs.html.scripts("client", routes.Assets.versioned(_).toString, name => getClass.getResource(s"/public/$name") != null)
```

Reply:
That would be a compile error. That code is within an html string, so it appears we need to insert a literal "$" into that line by using $$.

Reply:
I don’t follow. We don’t want the resulting HTML string to contain `name => getClass.getResource(s"/public/$name") != null`, we want to evaluate wether the resource is in our classpath or not.

Am I wrong?

Reply:
Hmm, tbh, I'm not sure! :-)
But `$name` is a compile error for sure. In my own very limited experience I've been constructing html in Play using the twirl stuff, like this:

```
@(title: String)(content: Html)(implicit flash: Flash)

<!DOCTYPE html>

<html>
    <head>
        <title>@title</title>
        <link rel="shortcut icon" type="image/png" href="@routes.Assets.versioned("images/favicon.png")">
        <!-- React -->
        <script src="https://unpkg.com/react@@16/umd/react.development.js"></script>
        <script src="https://unpkg.com/react-dom@@16/umd/react-dom.development.js"></script>
    </head>
    <body>
        @flash.get("error")
        @flash.get("success")
        @content
        @scalajs.html.scripts("client", routes.Assets.versioned(_).toString, name => getClass.getResource(s"/public/$name") != null)
    </body>
</html>
```

which would have a single `$` because this is not within a string. I took the passing local test as a green light that I'd done it 'correctly' in that the `$$` resolves to `$` and the script is executed correctly. Am I mistaken? I'm _very_  new to all of this so it's quite possible!

Reply:
I guess the test is insufficient because now I see that we should not use `@` but `$`:

```suggestion
            ${ scalajs.html.scripts("client", routes.Assets.versioned(_).toString, name => getClass.getResource(s"/public/$name") != null) }
```

That one should work.

Reply:
That is a compile error too, because the `routes.Assets` is not recognised, presumably because we haven't used twirl's magic `@` anywhere.

Reply:
Ah, it must be a missing import that is usually inserted by default by Twirl. Try with the fully qualified name: `controllers.routes.Assets.versioned`.

Reply:
Ok, that compiles and the test appears to pass locally.